addons/hw_drivers/iot_handlers/drivers/SerialBaseDriver.py

```diff
@@ -111,6 +111,9 @@ def _do_action(self, data):
             _logger.exception(msg)
             self._status = {'status': self.STATUS_ERROR, 'message_title': msg, 'message_body': traceback.format_exc()}
             self._push_status()
+        finally:
+            self._status = {'status': self.STATUS_CONNECTED, 'message_title': '', 'message_body': ''}
+            self._push_status()
```

Comment:
```suggestion
            self._status = {'status': self.STATUS_CONNECTED, 'message_title': '', 'message_body': ''}
```
Doesn't look like we need to send "connected" status to frontend every time at the end

Reply:
will fix SerialBlackBoxDriver in enterprise, so will be closing this PR